java/vector/src/main/codegen/templates/FixedValueVectors.java

```diff
@@ -611,7 +651,41 @@ public void setSafe(int index, int start, ArrowBuf buffer){
    }
 
    public void set(int index, int start, ArrowBuf buffer){
-     data.setBytes(index * ${type.width}, buffer, start, ${type.width});
+     data.setBytes(index * TYPE_WIDTH, buffer, start, TYPE_WIDTH);
+   }
+
+      <#elseif minor.class == "FixedSizeBinary">
+  public void set(int index, ${minor.class}Holder holder){
```

Comment:
These set methods looks similar to the existing ones, why do we need to have special code gen for these?